DH_Engine/Classes/DHVehicleCannonPawn.uc

```diff
@@ -205,10 +205,8 @@ simulated function array<DHArtillerySpottingScope.STargetInfo> PrepareTargetInfo
 
     WeaponLocation = VehWep.Location;
     WeaponLocation.Z = 0.0;
-    WeaponRotation.Yaw = CustomAim.Yaw;
-    // WeaponRotation.Yaw = class'UUnits'.static.MilsToUnreal(
-    //     class'UMath'.static.Floor(
-    //     class'UUnits'.static.UnrealToMils(CustomAim.Yaw), YawScaleStep));
+
+    WeaponRotation.Yaw = VehWep.Rotation.Yaw;
```

Comment:
This breaks the deflection readout for mortars. `VehicleWeapon` rotation is not utilized on mortars; it's always set to some arbitrary value.

Reply:
The deflection readout was completely off on mortars when i tested this.